docs/sdk/v4/guides/swaps/01-quoting.md

```diff
@@ -0,0 +1,117 @@
+
+export const CurrentConfig: SwapExactInSingle = {
+    poolKey: {
+        currency0: ETH_TOKEN.address,
+        currency1: USDC_TOKEN.address,
+        fee: FEE_AMOUNT_LOW,
+        tickSpacing: TICK_SPACING_SIXTY,
+        hooks: EMPTY_HOOK,
+    },
+    zeroForOne: true,
+    amountIn: ethers.utils.parseUnits('1', ETH_TOKEN.decimals).toString(), 
+    amountOutMinimum: "0",
+    hookData: '0x00'
+}
+```
+
+The pool used is defined by a pair of tokens in `constants.ts`.
+You can also change these two tokens and the other pool parameters in the config, just make sure a pool actually exists for your configuration.
+Check out the top pools on [Uniswap](https://app.uniswap.org/#/pools).
+
+```typescript
+import { Token } from '@uniswap/sdk-core'
+
+const ETH_TOKEN = new Token(
+  1,
+  '0x0000000000000000000000000000000000000000',
+  18,
+  'ETH',
+  'Ether'
+)
+
+const USDC_TOKEN = new Token(
+  1,
+  '0x89D9b1CD8A61D0581E22E04f1547B2Cc1232B848',
+  6,
+  'USDC',
+  'USD//C'
+)
+```
+
+## Referencing the Quoter contract and getting a quote
+
+To get quotes for trades, Uniswap has deployed a **Quoter Contract**. We will use this contract to fetch the output amount we can expect for our trade, without actually executing the trade.
+
+Now, we need to construct an instance of an **ethers** `Contract` for our Quoter contract in order to interact with it:
+
+```typescript
+const quoterContract = new ethers.Contract(
+  QUOTER_CONTRACT_ADDRESS,
+  Quoter.abi, // Import or define the ABI for Quoter contract
+  new ethers.providers.JsonRpcProvider("RPC") // Provide the right RPC address for the chain
+)
+```
+
+We get the `QUOTE_CONTRACT_ADDRESS` for our chain from [Uniswap Deployments](https://docs.uniswap.org/contracts/v4/deployments).
+
+We can now use our Quoter contract to obtain the quote.
+
+In an ideal world, the quoter functions would be `view` functions, which would make them very easy to query on-chain with minimal gas costs. However, the Uniswap V4 Quoter contracts rely on state-changing calls designed to be reverted to return the desired data. This means calling the quoter will be very expensive and should not be called on-chain.
+
+To get around this difficulty, we can use the `callStatic` method provided by the **ethers.js** `Contract` instances.
+This is a useful method that submits a state-changing transaction to an Ethereum node, but asks the node to simulate the state change, rather than to execute it. Our script can then return the result of the simulated state change:
+
+```typescript
+const quotedAmountOut = await quoterContract.callStatic.quoteExactInputSingle({
+    poolKey: CurrentConfig.poolKey,
+    zeroForOne: CurrentConfig.zeroForOne,
+    exactAmount: CurrentConfig.amountIn, 
+    hookData: CurrentConfig.hookData,
+})
+
+console.log(ethers.utils.formatUnits(quotedAmountOut, USDC_TOKEN.decimals));
+```
+
+The result of the call is the number of output tokens you'd receive for the quoted swap.
```

Comment:
lets avoid contractions since non-native English speakers are not as familiar

```suggestion
The result of the call is the number of output tokens you would receive for the quoted swap.
```